test/bigtest/network/factories/piece.js

```diff
@@ -5,8 +5,8 @@ import {
 
 export default Factory.extend({
   id: () => faker.random.uuid(),
-  poLineId: () => faker.random.uuid(),
-  poLineNumber: (id) => `${id}${faker.random.alphaNumeric()}${faker.random.alphaNumeric()}${faker.random.alphaNumeric()}`,
+  poLineId: 'e9009acb-5e89-40b7-8b07-6d565f567778',
```

Comment:
I don't think you need to use such constants in factories, you can easily pass on calling create if needed. Factories are exist to generate data, not to mock

Reply:
ok. got it.